### DISCUSSION TITLE
subfinder not start because go1.20 not install on the termux

### DISCUSSION THREAD

**Discussion by sapateira** [projectdiscovery/subfinder#1063] [General]
Hello, first of all, thank you very much for the fantastic tool. it helps me a lot. I'm not able to start with the subfinder since termux was transferred to go1.21. Can you help me to downgrade from go1.21 to 1.20 so I can then install the subfinder and use it as before. thanks

**Comment by MetzinAround** [comment]
Hi! thanks for the issue. 

To downgrade go, you have to delete it and reinstall the version you want. 

https://go.dev/doc/install#install

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
